src/chaptercleaner.test.ts

```diff
@@ -1,28 +1,324 @@
-import ChapterCleaner from './chaptercleaner.js';
-
 import { describe, expect, it } from 'vitest';
 
+import ChapterCleaner from './chaptercleaner.js';
+import { CleanChapterParams } from 'types.js';
+
 describe('ChapterCleaner', () => {
-  describe('keepOnlyBodyContents', () => {
```

Comment:
Did we loose the original tests, or are they not needed anymore?

Reply:
don't need, because of FORBID_TAGS
https://github.com/GevatterGaul/epubts/pull/7/files#diff-f0ffa60e6fd75bab77377cd8c1fe6a4bda716f3e4d7089452a55d7152e7e40b9R119